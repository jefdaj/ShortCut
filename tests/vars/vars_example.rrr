result = "vars example not written yet"
