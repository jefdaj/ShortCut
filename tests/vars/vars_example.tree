.
|-- cache
|-- exprs
|   `-- str
|       `-- b577bc3d8c.str -> ../../cache/lines/...
`-- vars
    `-- result -> ../exprs/str/b577bc3d8c.str

4 directories, 2 files
