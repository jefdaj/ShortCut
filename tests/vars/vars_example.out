"vars example not written yet"
